eos-tech-support/eos-diagnostics

```diff
@@ -53,6 +53,7 @@ function collectBoardInfo() {
     try {
         let [res, contents] = GLib.file_get_contents('/proc/device-tree/compatible');
         output += 'compatible: ' + contents;
+        output = output.replace(/\0/g, ' ');
```

Comment:
Well, I wonder if we should do this at the end of the function on the whole of `output`, unconditionally. Is it guaranteed that the DMI files don't contain embedded NULs?